docs/topics/OAuth2.md

```diff
@@ -386,6 +387,7 @@ Returns the bot's OAuth2 application info.
 | primary_sku_id?        | snowflake                                                  | if this application is a game sold on Discord, this field will be the id of the "Game SKU" that is created, if exists     |
 | slug?                  | string                                                     | if this application is a game sold on Discord, this field will be the URL slug that links to the store page               |
 | cover_image?           | string                                                     | if this application is a game sold on Discord, this field will be the hash of the image on store embeds                   |
+| flags                  | int                                                        | the application's public flags                                                                                            |
```

Comment:
Is there any specific reason why this isn't called `public_flags`?

Reply:
I think that all the flags on an application actually are public, as opposed to users where we specifically differentiate. Calling them the public flags covers my butt for now but I can confirm.

Reply:
It would be useful for future-proofing for a future where bots could be boosted.